lib/puppetserver/ca/utils/file_system.rb

```diff
@@ -50,6 +50,17 @@ def self.check_for_existing_files(one_or_more_paths)
           errors
         end
 
+        def self.symlink_to_old_cadir(current_cadir)
```

Comment:
lmk if there's a better shared place to put this, nothing seemed like an obviously good place

Reply:
I really want to make this just a forceful symlink function and if it really needs to hardcode the ca defaults then move it to the `local_certificate_authority` but as is matches pretty well with how this module is structured. It _might_ be time to refactor it (the local ca class didn't exist when we made the file_system module) but I'm okay waiting a bit before doing a refactor.

Reply:
I could make this more generic, but I think we'd still need another new method wrapped around it because we need to check whether we're using the new default or not before trying to link. Do you think that's worthwhile?